src/client/components/Admin/TableOrders/RefactorOrders.jsx

```diff
@@ -98,34 +98,34 @@ class RefactorOrders extends React.Component {
         </div>
       )
     })
-    const table = <React.Fragment> <div className='table-orders__title'>Table orders</div>
-      <div className='table-orders__table'>
-        <table>
+    const table = <React.Fragment> <div className='table-model__title'>Table orders</div>
+      <div className='table-model__container'>
+        <table className='table-model__container__table'>
           <tbody>
             <tr>
               <th>№</th>
               <th>Client name</th>
               <th>Client email</th>
-              <th>Type clock</th>
-              <th>City</th>
-              <th>Date</th>
-              <th>Master</th>
+              <th className='text-to-center'>Type clock</th>
```

Comment:
.table-model__container__table th, .table-model__container__table td {
    text-align: center;
}

Implement in less style